async-shim.js

```diff
@@ -1,19 +0,0 @@
-var compat = require('./browser')
```

Comment:
I'm not against the renaming of this file, but it does make the diff harder to see, can we have the name changed back and then we can do a separate pull to rename the file?